### PR TITLE
GUACAMOLE-446: Documentation for new drive-name parameter.

### PR DESCRIPTION
**Pull request by necouchman**
Documentation for the parameter allowing redirected drive name to be configured.

### CODE REVIEW

File: src/chapters/configuring.xml [hunk 1]
```diff
@@ -2059,6 +2059,18 @@ guaclog: INFO: All files interpreted successfully.</computeroutput>
...
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>drive-name</parameter></entry>
+                                <entry>
+                                    <para>The name of the filesystem used when passed through to
+                                        the RDP session.  This is the name that users will see in their
+                                        Computer/My Computer area along with client name (for example,
+                                        "Guacamole on Guacamole RDP"), and is also the name of the share
+                                        when accessing the special \\tsclient network location.</para>
```
Comment (mike-jumper) [inline src/chapters/configuring.xml]: `\\tsclient` should be wrapped in a [`<filename>` tag](https://tdg.docbook.org/tdg/5.0/filename.html).

**Comment by mike-jumper** [comment]
Please also update the merge base to `staging/1.0.0`.

File: src/chapters/configuring.xml (hunk 1 above)
Comment (necouchman) [reply src/chapters/configuring.xml]: Tagged.

**Comment by necouchman** [comment]
Merge base changed to `staging/1.0.0`.

**Pull request merged**